packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -15,22 +16,37 @@ const FloatingRate = ({
     error_messages,
     fiat_currency,
     local_currency,
+    onChange,
     offset,
-    placeholder,
     ...props
 }) => {
     const { general_store } = useStores();
     const { name, value, required } = props;
-    const market_feed = value ? parseFloat(exchange_rate * (1 + value / 100)).toFixed(2) : exchange_rate;
+    // Get market feed restricted to 2 decimal places
+    const market_feed = value ? truncateDecimal(parseFloat(exchange_rate * (1 + value / 100)), 2) : exchange_rate;
 
+    const onBlurHandler = e => {
+        let float_rate = e.target.value;
+        if (float_rate && float_rate.length) {
```

Comment:
Why do we need to check if `float_rate` has length?

Reply:
The input field is a text field. I am preventing the function body statements from execution if the user has just provided spaces as input

Reply:
Correct me if I'm wrong, but don't spaces count in length too?

Reply:
Yes it does. My bad. I will change it to `float_rate.trim().length`

Reply:
Sorry for late review, but still the same question remains: why do we need to check if `float_rate` has length if it exists and why do you trim it?